test/transmuxer/transmuxer_integration.js

```diff
@@ -325,6 +325,23 @@ describe('Transmuxer Player', () => {
       await player.unload();
     });
 
+    it('H.264+AAC in TS with rollover', async () => {
+      // eslint-disable-next-line max-len
+      await player.load('/base/test/test/assets/hls-ts-rollover/playlist.m3u8');
+      await video.play();
+      expect(player.isLive()).toBe(false);
+
+      // Wait for the video to start playback.  If it takes longer than 10
+      // seconds, fail the test.
+      await waiter.waitForMovementOrFailOnTimeout(video, 10);
+
+      // Play for 15 seconds, but stop early if the video ends.  If it takes
```

Comment:
I see 20s of content, and we play for 15s.  Where in that stream does the rollover occur?  What is the behavior of the test if the rollover isn't handled?

These questions should be answered in comments in the test.

Reply:
I added a comment.

The purpose of the test is to check if the video plays well with rollover.

Without rollover:


https://github.com/shaka-project/shaka-player/assets/8983024/3c7e537f-bce9-41ca-9e52-32bca0ced9d5


With rollover:


https://github.com/shaka-project/shaka-player/assets/8983024/b8d02724-bfce-47e1-a811-78f73d1f7a2b